### PR TITLE
Branch perf v13 ttl

### PR DESCRIPTION
**Pull request by yarongilor**
feature(Alternator): Add a performance test for TTL
	In order to get a performance report for
	Alternator new TTL feature.

This PR is needed since cannot backport from the original master PR.
The corresponding master PR with more details is: https://github.com/scylladb/scylla-cluster-tests/pull/5269

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
Started to test in: https://jenkins.scylladb.com/job/scylla-staging/job/yarongilor/job/scylla-master-perf-regression-alternator-throughput-test-ttl/

**Comment by fruch** [comment]
Can you explain what's the PR is for ?

Why can't you wait for the TTL to be merged to master ?

why can't you write performance test with master ?

**Comment by roydahan** [comment]
This PR is only for testing, not to be merged.

File: sdcm/cluster_docker.py [hunk 1]
```diff
@@ -88,6 +92,11 @@ def _get_public_ip_address(self) -> Optional[str]:
     def _get_private_ip_address(self) -> Optional[str]:
         return self.public_ip_address
 
+    def _refresh_instance_state(self):
```
Comment (fgelcer) [inline sdcm/cluster_docker.py:95]: it is more `refresh_instance_ip_addresses` than state

File: sdcm/utils/docker_remote.py [hunk 2]
```diff
@@ -96,7 +115,7 @@ def _refresh_instance_state(self):
     def check_spot_termination(self):
         pass
 
-    @property
+    @ property
```
Comment (fgelcer) [inline sdcm/utils/docker_remote.py:118]: isn't this space breaking the property decorator?

**Comment by yarongilor** [comment]
@fruch , master would be much better if it is accepted to use. just have to confirm that. we can run master with scylla 5.1 ttl vs. original alternator. @roydahan , please confirm moving to master.

**Comment by roydahan** [comment]
Confirmed.
At the maximum you can run the same without TTL to be sure it’s comparable.

**Comment by yarongilor** [comment]
Closing this one. moved the new pipeline files to the original PR https://github.com/scylladb/scylla-cluster-tests/pull/5269

**Pull request closed without merging**